gdx/src/com/badlogic/gdx/utils/JsonMatcher.java

```diff
@@ -136,6 +158,14 @@ public class JsonMatcher extends JsonSkimmer {
 	final IntArray path = new IntArray();
 	Pattern processPattern;
 
+	public JsonMatcher () {
+	}
+
+	public JsonMatcher (String... patterns) {
```

Comment:
Maybe some docs here would be nice? I'm guessing multiple patterns are all tried as if they were separated by OR, so any matches get yielded, but I could also see this producing each pattern's output in turn.

Reply:
It just adds all the patterns. I'll add to class docs, "When multiple patterns are added they match in parallel, each with independent capture and processing." Also, about `reject()`, "A pattern can reject a different pattern."